<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AnatomyDefine</class>
 <widget class="QDialog" name="AnatomyDefine">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>750</width>
    <height>550</height>
   </rect>
  </property>
  <property name="palette">
   <palette>
    <active>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </active>
    <inactive>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>255</red>
        <green>0</green>
        <blue>0</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>204</red>
        <green>232</green>
        <blue>207</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </inactive>
    <disabled>
     <colorrole role="Text">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>120</red>
        <green>120</green>
        <blue>120</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Base">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
     <colorrole role="Window">
      <brush brushstyle="SolidPattern">
       <color alpha="255">
        <red>160</red>
        <green>160</green>
        <blue>160</blue>
       </color>
      </brush>
     </colorrole>
    </disabled>
   </palette>
  </property>
  <property name="windowTitle">
   <string>AnatomyDefine</string>
  </property>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>10</y>
     <width>20</width>
     <height>241</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>270</y>
     <width>20</width>
     <height>221</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>270</y>
     <width>20</width>
     <height>231</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>250</y>
     <width>721</width>
     <height>16</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="QListWidget" name="lwSkin">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSkinAdd">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSkinSub">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QListWidget" name="lwMuscle">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwBone">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>40</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwOrgan">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>300</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwArtery">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>300</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QListWidget" name="lwNerve">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>300</y>
     <width>191</width>
     <height>151</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMuscleSub">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnMuscleAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoneAdd">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnBoneSub">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>200</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrganSub">
   <property name="geometry">
    <rect>
     <x>50</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnOrganAdd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNerveSub">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnNerveAdd">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArteryAdd">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/add.png</normaloff>:/MyImages/images/add.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QPushButton" name="btnArterySub">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>460</y>
     <width>35</width>
     <height>35</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="icon">
    <iconset resource="anatomyshow.qrc">
     <normaloff>:/MyImages/images/sub.png</normaloff>:/MyImages/images/sub.png</iconset>
   </property>
   <property name="iconSize">
    <size>
     <width>35</width>
     <height>35</height>
    </size>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>54</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>皮肤层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>10</y>
     <width>71</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>肌肉层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>10</y>
     <width>71</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>骨骼层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>260</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>器官层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>260</y>
     <width>54</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>神经层</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>500</x>
     <y>270</y>
     <width>61</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>血管层</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btnGenerateModel">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>510</y>
     <width>111</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>生成模型</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>520</y>
     <width>171</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>24</number>
   </property>
  </widget>
  <widget class="QLabel" name="lblShowMsg">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>515</y>
     <width>411</width>
     <height>21</height>
    </rect>
   </property>
   <property name="palette">
    <palette>
     <active>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </active>
     <inactive>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </inactive>
     <disabled>
      <colorrole role="Text">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ButtonText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>120</red>
         <green>120</green>
         <blue>120</blue>
        </color>
       </brush>
      </colorrole>
      <colorrole role="ToolTipText">
       <brush brushstyle="SolidPattern">
        <color alpha="255">
         <red>255</red>
         <green>0</green>
         <blue>0</blue>
        </color>
       </brush>
      </colorrole>
     </disabled>
    </palette>
   </property>
   <property name="font">
    <font>
     <pointsize>12</pointsize>
    </font>
   </property>
   <property name="text">
    <string>现在可以观看模型了！点击左边“anatomy” 显示即可！</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="anatomyshow.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>btnSkinAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>SkinAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnSkinSub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>SkinSub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMuscleAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>MuscleAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnMuscleSub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>MuscleSub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoneAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>BoneAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>201</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnBoneSub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>BoneSub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>231</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOrganAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>OrganAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnOrganSub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>OrganSub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>37</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNerveAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>NerveAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnNerveSub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>NerveSub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>267</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnArteryAdd</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>ArteryAdd()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>451</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnArterySub</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>ArterySub()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>537</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>btnGenerateModel</sender>
   <signal>clicked()</signal>
   <receiver>AnatomyDefine</receiver>
   <slot>GenerateModel()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>255</x>
     <y>525</y>
    </hint>
    <hint type="destinationlabel">
     <x>374</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>SkinAdd()</slot>
  <slot>SkinSub()</slot>
  <slot>MuscleAdd()</slot>
  <slot>MuscleSub()</slot>
  <slot>BoneAdd()</slot>
  <slot>BoneSub()</slot>
  <slot>OrganAdd()</slot>
  <slot>OrganSub()</slot>
  <slot>NerveAdd()</slot>
  <slot>NerveSub()</slot>
  <slot>ArteryAdd()</slot>
  <slot>ArterySub()</slot>
  <slot>NextStep()</slot>
  <slot>GenerateModel()</slot>
 </slots>
</ui>
